site/en/blog/immutable-document-domain/index.md

```diff
@@ -107,13 +107,12 @@ The [current discussion with other browser
 vendors](https://github.com/w3ctag/design-reviews/issues/564) is moving in
 the same direction.
 
-For example, if a hosting service provides different subdomains per user, an
-attacker can set `document.domain` to pretend they are the same-origin
-as another user's page. Further, an attacker can host a website under a shared
-hosting service, which serves sites through the same IP address with different
-port numbers. In that case, the attacker can pretend to be on the same-site-but-same-origin
-as yours. This is possible because `document.domain` ignores the port number
-part of the domain.
+For example, when two pages set `document.domain`, they can pretend as if they
+are the same-origin. This is particularly critical when using a shared hosting
+service with different subdomains per user, setting `document.domain` opens up
+an access to any other site hosted by the same service, making it easily
+accessible to any attackers. This is possible because `document.domain` ignores
+the port number part of the domain.
```

Comment:
```suggestion
For example, when two pages set `document.domain`, they can pretend as if they
are the same-origin. This is particularly critical when these pages use a shared
hosting service with different subdomains. Setting `document.domain` opens up
access to all other sites hosted by that same service, which makes it easier for
attackers to access your sites. This is possible because `document.domain`
ignores the port number part of the domain.
```